.gitignore

```diff
@@ -22,6 +22,7 @@ vendor/
 !.vscode/extensions.json
 
 *.env
+manifests/
```

Comment:
should this be `kubernetes/manifests/` ?

Reply:
yeah